### PR TITLE
DEX-919 Swagger errors

### PR DESCRIPTION
**Pull request by vsuharnikov**

### CODE REVIEW

File: dex/src/main/scala/com/wavesplatform/dex/api/http/entities/HttpMarketDataWithMeta.scala [hunk 1]
```diff
@@ -8,20 +8,25 @@ case class HttpMarketDataWithMeta(
     @ApiModelProperty(
       value = "Base58 encoded amount asset ID",
       dataType = "string",
-      example = "8LQW8f7P5d5PZM7GtZEBgaqRPGSzS3DfPuiXrURJ4AJS"
+      example = "8LQW8f7P5d5PZM7GtZEBgaqRPGSzS3DfPuiXrURJ4AJS",
+      required = true
```
Comment (DenisSerebryansky) [inline dex/src/main/scala/com/wavesplatform/dex/api/http/entities/HttpMarketDataWithMeta.scala:12]: Why? This is response, not request

File: dex/src/main/scala/com/wavesplatform/dex/api/http/entities/HttpMarketStatus.scala [hunk 2]
```diff
@@ -6,37 +6,18 @@ import com.wavesplatform.dex.model.{LastTrade, LevelAgg}
...
                             @ApiModelProperty(
                               value = "Side (sell or buy)",
                               dataType = "string",
-                              example = "buy",
-                              allowEmptyValue = true
+                              example = "buy"
                             )
                             lastSide: Option[OrderType],
-                            @ApiModelProperty(
-                              allowEmptyValue = true,
```
Comment (DenisSerebryansky) [inline dex/src/main/scala/com/wavesplatform/dex/api/http/entities/HttpMarketStatus.scala:25]: This was in warnings?

**Review by DenisSerebryansky — CHANGES_REQUESTED** [review]

File: dex/src/main/scala/com/wavesplatform/dex/api/http/entities/HttpMarketStatus.scala (hunk 2 above)
Comment (vsuharnikov) [reply dex/src/main/scala/com/wavesplatform/dex/api/http/entities/HttpMarketStatus.scala:25]: Yep

File: dex/src/main/scala/com/wavesplatform/dex/api/http/entities/HttpMarketDataWithMeta.scala (hunk 1 above)
Comment (vsuharnikov) [reply dex/src/main/scala/com/wavesplatform/dex/api/http/entities/HttpMarketDataWithMeta.scala:12]: https://swagger.io/specification/#schema-object
```
The Schema Object allows the definition of input and output data types
...
For more information about the properties, see JSON Schema Core and JSON Schema Validation
```
https://tools.ietf.org/html/draft-wright-json-schema-validation-00#section-5.15 :
```
An object instance is valid against this keyword if its property
 set contains all elements in this keyword's array value.
```
As I understood, responses are validated too.

**Review by DenisSerebryansky — APPROVED** [review]

**Pull request merged**